tools/cell_census_builder/experiment_builder.py

```diff
@@ -70,29 +78,11 @@ class ExperimentBuilder:
     anndata_cell_filter_spec: AnnDataFilterSpec
     gene_feature_length_uris: List[str]
     gene_feature_length: pd.DataFrame
-    build_state: "ExperimentBuilder.BuildState"
-
-    # builder state sanity check, used to catch usage errors.
 
-    class BuildState(IntEnum):
-        Initialized = 0
-        Created = 1
-        AxisWritten = 2
-        X_Created = 3
-        X_JoinIdMetadataCreated = 4
-        X_Written = 5
-        X_Presence_Written = 6
-
-        def next(self) -> "ExperimentBuilder.BuildState":
-            return ExperimentBuilder.BuildState(self.value + 1)
```

Comment:
Due to refactoring, it was going to take work to make this work again; I opted to remove this state machine.  Useful for sanity checks, but the validator is a reasonable check now that census is built correctly. I can be persuaded to reinstate.